test_data/third_party/2and3/six_test.py

```diff
@@ -0,0 +1,17 @@
+import pip  # type: ignore
+
+pip.main('install six==1.10.0'.split())
```

Comment:
It would be nice if the `runtime_test.py` driver can somehow take care of dependencies, maybe through some sort of magical comment that it scans for.